### ISSUE TITLE
Bookbuilding View

### ISSUE THREAD

**Issue by SergejSoft** [Neufund/platform-frontend#1293]
Bookbuilding views. 

<img width="1311" alt="screen shot 2018-09-05 at 17 33 48" src="https://user-images.githubusercontent.com/2290474/45104419-5eb1ec80-b132-11e8-9f56-88454703984c.png">

Public View>Bookbuilding View

In this view we have a button Back now. 
https://projects.invisionapp.com/d/main#/projects/prototypes/15287103

Feature list: 
Investor can pledge investment amount. 
Investor see total number of investors 
investor see total number of pledget capital.
investor can delete and update his pledge. 

@rudolfix marcin, we need end points for database entries. Here is the swager for Pledges API
https://platform.neufund.io/api/eto-listing/ui/#/Pledge

**Comment by mpneu** [comment]
https://app.zeplin.io/project/5a8a92c89c1a166a6a6e8f37?seid=5b8fdd3278b3ba5559a1e638

**Comment by desfero** [comment]
Blocked by the backend. https://github.com/Neufund/platform-backend/issues/980

**Comment by glupiawymowka** [comment]
Please fit the button 
![image](https://user-images.githubusercontent.com/18315589/45482648-ed37f680-b74e-11e8-8713-d24c7a921a29.png)

Doesn't look like a button (see cursor)
![image](https://user-images.githubusercontent.com/18315589/45483934-b5cb4900-b752-11e8-89a9-252b6a01c8b9.png)

Can we improve the wording here? It sounds a bit too technical. @meneramandarin could you take a look? I think that something like "The minimum value is ..." would be enough
![image](https://user-images.githubusercontent.com/18315589/45486699-52deaf80-b75c-11e8-89d6-2b664f020398.png)

 To sum up the overall functionality: 
Bookbuilding impossible when the user didn't verify email/KYC -> OK
"Show my email" option saves after refresh -> OK
amount can be correctly added, changed and deleted -> OK

**Comment by meneramandarin** [comment]
Agreed. 
"The minimum amount is 100€" sounds good to me.

**Comment by desfero** [comment]
@glupiawymowka 
1. The button is fixed.
2. The non-interactive elements are a part of https://github.com/Neufund/platform-frontend/issues/1343
3. Error messages are a part of https://github.com/Neufund/platform-frontend/issues/1357
3.
